config/locales/en.yml

```diff
@@ -51,8 +51,22 @@ en:
           We just wanted to let you know that you have %{total} new invitation(s),
           which can be used to create new accounts at the archive. You can invite a
           friend at %{invitation_page}.
-          Cheers, 
+          Cheers,
           %{archive_name}
+    admin_hidden_work_notification:
+      subject: "[%{app_name}] Your work has been hidden by the Abuse Team"
+      hello: "Dear %{user},"
+      tos: "Terms of Service"
+      abuse: "contact our Abuse Team"
+      sincerely: "Sincerely,"
+      ao3_abuse: "AO3 Abuse"
+      part2: "We are investigating the matter and will contact you once the investigation has been completed. While the work is under investigation it has been hidden; only you will be able to access it."
+      html:
+        part1: "Your work <i>%{pretty_title}</i> has been reported as being in violation of the Archive of Our Own's %{tos}."
```

Comment:
`%{pretty_title}` is just supposed to be `%{title}`, since that's the name of the key. Also, the `<i>` and `</i>` should be removed here.